lib/perlfaq4.pod

```diff
@@ -1116,6 +1116,43 @@ signal that I missed something:
 
     print $string;
 
+
+=head2 Does Perl have anything like Ruby's #{} or Python's f string?
+
+Unlike the others, Perl allows you to embed a variable naked in a quoted
+string C<"variable $variable">, when there isn't whitespace around the
```

Comment:
Pedantically, it's not whitespace that's required to delimit a variable name in a double-quoted string, but any character that isn't allowed in a variable name, i.e. anything that matches `\W`.

Reply:
Whitespace /braces are a stylistic rule, explanation of what should be expected to parse is tangental. I think I can insert "for readability" when there ... to be both correct and avoid the tangent.